libs/partners/milvus/langchain_milvus/vectorstores/milvus.py

```diff
@@ -298,8 +319,30 @@ def __init__(
         self._primary_field = primary_field
         # In order for compatibility, the text field will need to be called "text"
         self._text_field = text_field
+
+        if isinstance(self.embedding_func, list):
+            if len(self.embedding_func) < 2:
+                logger.error(
+                    "At least two embeddings functions"
+                    "are required for multi-vector search."
+                )
+                raise ValueError(
+                    "Multi-vector search requires at least two embeddings functions."
+                )
+            if not isinstance(vector_field, list):
+                vector_field = [
+                    f"vector_{i+1}_{e.__class__.__name__}"
+                    for i, e in enumerate(self.embedding_func)
+                ]
```

Comment:
```suggestion
                vector_field = [
                    f"vector_{i}"
                    for i in range(len(self.embedding_func))
                ]
```
I think the extra class name is maybe easier for debugging but harder for usage.

Reply:
Got it